### ISSUE TITLE
video-caption rule should not have excludeHidden: false

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3488]
These two rules should ignore hidden elements. The following should not be flagged as incomplete:

https://act-rules.github.io/rules/eac66b#inapplicable-example-1
```html
<video src="/test-assets/perspective-video/perspective-video.mp4" controls style="display: none;"></video>
```

The only thing to do on this is update lib/rules/video-caption.json and remove excludeHidden, and possibly tweak the integration tests for it.

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, when `controls style="dispaly: none;" ` , `vidoe-caption` is rule is going to `inapplicable`
Script: ` <video src="/Users/dq_user/downloads/video.mp4" controls style="display: none;"></video>`

![Image](https://user-images.githubusercontent.com/47409579/187365817-b20d5e90-0279-4b2c-b983-3dd3c847f0e5.png)
